Review of ARRM (#40)
w3c/wai-arrm: content/index.md

{% include_cached box.html type="end" %}
{:/}

> ARRM includes a table of high-level WCAG requirements — called "success criteria" — and a table of tasks that address the WCAG success criteria. The tables include primary, secondary, and contributor responsibilities.

Reviewer: ```suggestion
ARRM includes a table of WCAG requirements — called "success criteria" — and a table of tasks that address the WCAG success criteria. The tables include primary, secondary, and contributor responsibilities.
```

## Typical roles and responsibilities

ARRM provides one approach for defining roles, tasks, and responsibilities. You can use these as they are, without any work to customize them.

* **[Roles Involved in Accessibility](roles)**
* **[WCAG Success Criteria](wcag-sc)** shows role responsibilities

> * **[Tasks Involved in Accessibility](tasks)** shows tasks you can do to help meet WCAG and role responsibilities — subsets of the tasks are listed for the different roles:

Reviewer: ```suggestion
* **[Tasks Involved in Accessibility](tasks)** shows tasks you can use to help meet WCAG and their role responsibilities — subsets of the tasks are listed for these roles:
```
